### PR TITLE
Add documentation in README

### LINKED ISSUE #6: Add documentation in README

### PR DESCRIPTION
**Pull request by gmargaritis**
Resolves #6

### CODE REVIEW

**Comment by gmargaritis** [comment]
@parisk 

1. Should we keep the `Glossary` section in the README?
2. Do you think that we should keep each function signature?
3. Should we also include our generated documentation? e.g. descriptions for parameters etc

File: README.md [hunk 1]
```diff
@@ -1,5 +1,369 @@
...
+| **Original API code**  | **Original help text**   | **Translated**     |
+|------------------------|--------------------------|--------------------|
+| `ΜΕ`                   | ΜΗ ΕΡΓΑΣΙΑ               | `ABSENT`           |
+| `ΑΝ`                   | ΑΝΑΠΑΥΣΗ/ΡΕΠΟ            | `REST_DAY`         |
+| `ΤΗΛ`                  | ΤΗΛΕΡΓΑΣΙΑ               | `WORK_FROM_HOME`   |
+| `ΕΡΓ`                  | ΕΡΓΑΣΙΑ                  | `WORK_FROM_OFFICE` |
+
+## License
+
+This project is licensed under the [`MIT License`](LICENSE)
```
Comment (parisk) [inline README.md:369]:
```suggestion
This project is licensed under the [`MIT License`](LICENSE)

---

<p align="center">
  <i>🦄 Built with <a href="https://withlogic.co/">LOGIC</a>. 🦄</i>
</p>
```

**Review by parisk — CHANGES_REQUESTED** [review]
Great work! A few comments and we should be good to go.

**Review by parisk — APPROVED** [review]
LGTM 🦄

**Pull request merged**